docs/release_notes.rst

```diff
@@ -9,6 +9,10 @@ v2024.X.x (2024-XX-XX)
 New Data Coverage
 ^^^^^^^^^^^^^^^^^
 
+EIA 923
+~~~~~~~
+* Added EIA 923 final release data from 2023. See :issue:`3684` and PR :pr:`3871`.
+
```

Comment:
This is the EIA-860 right? Maybe you had 923 on the brain this morning with the archiver false alarm.

Reply:
Yes, 100% a foolish copy-paste error I'll fix!